src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Mappers/Apprentice/InformViewModelMapperTests.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Generic;
```

Comment:
I think the class name should be changed - as this contains tests for ChangeEmployerRequestDetailsViewModel